modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/package.json

```diff
@@ -4,8 +4,7 @@
 	"scripts": {
 		"build": "liferay-npm-scripts build",
 		"checkFormat": "liferay-npm-scripts check",
-		"format": "liferay-npm-scripts fix",
-		"test": "liferay-npm-scripts test"
+		"format": "liferay-npm-scripts fix"
```

Comment:
What's the reason for removing the `test` script? Is it useless?

Reply:
It was breaking CI. It tried to run these tests and there were none.

Reply:
Ok so the reason is because this has been copy pasted and we have no tests, so we don't need it.
This PR is not horrible :heart: